docs/user-guide/rules.md

```diff
@@ -20,6 +20,7 @@ Disallow deprecated things with these `no-deprecated` rules.
 | :-- | :-: | :-: |
 | [`at-rule-no-deprecated`](../../lib/rules/at-rule-no-deprecated/README.md)<br/>Disallow deprecated at-rules. | ✅ | 🔧 |
 | [`declaration-property-value-keyword-no-deprecated`](../../lib/rules/declaration-property-value-keyword-no-deprecated/README.md)<br/>Disallow deprecated keywords for properties within declarations. | ✅ | 🔧 |
+| [`media-type-no-deprecated`](../../lib/rules/media-type-no-deprecated/README.md)<br/>Disallow deprecated media types. | ✅ | |
```

Comment:
@jeddy3

We will have to schedule a new `stylelint-config-recommended` version.

Reply:
SGTM. Let's see if we can land https://github.com/stylelint/stylelint/pull/8669 in the next release, too. So that we can include both new rules in the config.